user/views.py

```diff
@@ -25,31 +25,55 @@ class RegisterAPIView(APIView):
     
     def post(self, request, format=None):
         data = request.data
-        #email = request.data.get('email')
-        #password = request.data.get('password')
+        data['authors'] = []
+        data['books'] = []
         serializer = MyUserSerializer(data=data)
         
         serializer.is_valid(raise_exception=True)
         user = serializer.save()
         user.set_password(data['password'])
         user.save()
-        response = Response()
-
-        response.data = {
+        
+        return Response({
             'status' : status.HTTP_201_CREATED,
             'message': 'User Registered Successfully',
-           }
-
-        return response
+          })
 
 class LoginAPIView(APIView):
     
-    def post(self, request, format=None):  # sourcery skip: use-named-expression
+    def post(self, request, format=None): 
+         
+        email = request.data.get("email")
+        password = request.data.get("password")
+        try:
+            user = authenticate(username=email, password=password)
+            token, created  = Token.objects.get_or_create(user=user)
+        except Exception:
```

Comment:
This error handling is wrong. Do it like yesterday, Just need to create the token inside if block.